### ISSUE TITLE
[Ubuntu 18.04] Movie recording broken (and GUI unreadable)

### ISSUE THREAD

**Issue by wxmerkt** [Gepetto/gepetto-viewer#92]
I just installed `gepetto-gui` from robotpkg and used the `Movie record` button - however, the status window looks like this:

![image](https://user-images.githubusercontent.com/1664508/57532943-88ac5600-7334-11e9-867a-aeeb50535a20.png)

On 18.04, `avconv` has become `ffmpeg` again - this may be the source of this issue?

Unrelated, the menu entries are all illegible until hovered

**Comment by jmirabel** [comment]
If avconv command is not available, recording a movie won't work. However,
I am surprised you didn't get any warning or error message. You can go to
/tmp/gepetto-gui/ to get the images.

About the menu entries, I have no idea what's going on. Something to do
with Qt...

BTW, you can install doxygen-icon-theme to get proper icons for some
buttons.

**Comment by nim65s** [comment]
gepetto-viewer from robotpkg does not yet comes with PythonQt.

@jmirabel : you told me that this would make some optional features unavailable. Maybe this is one of those ?

**Comment by wxmerkt** [comment]
> If avconv command is not available, recording a movie won't work. However, I am surprised you didn't get any warning or error message. You can go to /tmp/gepetto-gui/ to get the images.

I aliased avconv=ffmpeg and it didn't work - there was neither an error message nor any output at all. I manually converted it to a movie later on.

> About the menu entries, I have no idea what's going on. Something to do with Qt...

FWIW, I only tried the Qt4 version, so it may vary on Qt5.

>  BTW, you can install doxygen-icon-theme to get proper icons for some buttons.

If this is required, shouldn't this be a dependency so that it auto-installs?

**Comment by jmirabel** [comment]
> gepetto-viewer from robotpkg does not yet comes with PythonQt.

This does not use PythonQt.

**Comment by jmirabel** [comment]
> If this is required, shouldn't this be a dependency so that it auto-installs?

It could. Probably, @nim65s didn't know it.

**Comment by jmirabel** [comment]
> I aliased avconv=ffmpeg and it didn't work - there was neither an error message nor any output at all. I manually converted it to a movie later on.

A bash alias won't work. The command isn't run in a bash shell. Can you try to add a symlink in one of the directories of PATH ?

**Comment by jmirabel** [comment]
Solved by #94 

> Unrelated, the menu entries are all illegible until hovered

You can create a Qt style sheet `stylesheet.txt`:
```
QMenu {
 background: white;
 color: black;
}
QMenu:selected {
 background: blue;
 color: white;
}
QToolTip {
 background: yellow;
 border: 1px solid black;
 color: black;
}
```
and run `gepetto-gui -stylesheet stylesheet.txt`
